packages/mdc-slider/README.md

```diff
@@ -89,6 +98,51 @@ DOM.
 </div>
 ```
 
+### Using a step value
+
+> **NOTE**: If a slider contains a step value it does _not_ mean that the slider is a "discrete"
+> slider. "Discrete slider" is a UX treatment, while having a step value is behavioral.
+
+`MDCSlider` supports quantization by allowing users to supply a floating-point `step` value via a
+`data-step` attribute.
+
+```html
+<div class="mdc-slider" tabindex="0" role="slider"
+     aria-valuemin="0" aria-valuemax="100" aria-valuenow="0"
+     data-step="2" aria-label="Select Value">
+  <!-- ... -->
+</div>
+```
+
+When a step value is given, the slider will quantize all values to match that step value, _except_
+for the minimum and maximum values, which can always be set. This is to ensure consistent behavior.
+
+The step value can be any positive floating-point number, or `0`. When the step value is `0`, the
+slider is considered to not have any step. A error will be thrown if you are trying to set step
+value to be a negative number.
+
+Discrete sliders are required to have a positive step value other than 0. If a step value of 0 is
+provided, or no value is provided, the step value will default to 1.
+
+### Display tick marks (discrete slider only)
+
+Discrete sliders support display tick marks on their tracks by adding `<div class="mdc-slider__tick-mark-set"></div>`
+to the track element.
```

Comment:
Add: "after the `mdc-slider__track-fill` element" (since it won't work if users try to add the tick mark set before the track fill)

Reply:
👍